### DISCUSSION TITLE
Script-Node Connections GUI

### DISCUSSION THREAD

**Discussion by Lawncoder** [godotengine/godot-proposals#9783] [Editor]
So I had an idea for Godot, as someone who has been using the engine for 3 days. Knowing what script connects to what can sometimes be a little confusing. And making changes to script connections can be a nightmare sometimes. Since I am a little helpless and unable to code anything myself, it would be pretty cool to add a window to scenes that shows boxes for all relevant nodes and scripts in the scene, and shows connections between scripts attached to nodes. The GUI would be really simple.

**Comment by AThousandShips** [comment]
You can easily find the connected scripts with the little icons next to the nodes, it also has a tooltip showing the path to the script when hovered
